src/buza/templates/buza/question_form.html

```diff
@@ -9,12 +9,23 @@
 {% block title %}Ask a question - {{ block.super }}{% endblock %}
 
 {% block content %}
+<div class="question__form_container">
     <form method="post"
           action="{% if question.pk %}{% url 'question-update' pk=question.pk %}{% else %}{% url 'question-create' %}{% endif %}">
+     <h2>Add a new Question</h2>
         {% csrf_token %}
-        {{ form.as_p }}
-        <p>
+       {% for field in form %}
+            <div>
+            <br/>{{ field.errors }}
+            {{ field.label_tag }}
+                <p class="question__form_helptext">
+                    {{ field.help_text }}</p>
+            {{ field }}
+            </div>
+        {% endfor %}
+         <br/><p>
```

Comment:
We should probably switch to `crispy-forms` here, rather than doing manual template formatting.